### PR TITLE
fix: add uiSchema type

### PR DESCRIPTION
**Pull request by shanejonas**

### CODE REVIEW

File: src/UISchema.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+export interface IUISchema {
+  appBar: {
+    ["ui:title"]: string,
+    ["ui:logoUrl"]: string,
+    ["ui:inputPlaceholder"]: string,
+    ["ui:splitView"]: boolean,
+  };
```
Comment (shanejonas) [inline src/UISchema.tsx]: hmmm, `appBar` really isnt part of this repo. should rethink this

**Comment by BelfordZ** [comment]
lmk when the conlfict are fixed, but lookin good

File: src/UISchema.tsx (hunk 1 above)
Comment (shanejonas) [reply src/UISchema.tsx]: @BelfordZ ^^ thoughts? should it be like [k: string]: any or something to allow `playground` to have another `IUISchema` that has `appBar` in it?

**Comment by shanejonas** [comment]
need to rethink this

**Pull request closed without merging**